guides/micronaut-scope-types/micronaut-scope-types.adoc

```diff
@@ -48,9 +48,7 @@ callout:singleton[1]
 
 === Singleton Test
 
-To use the testing features described in this section, add the following dependency to your build file:
-
-dependency:junit-jupiter-params[groupId=org.junit.jupiter,scope=testImplementation]
+common:scopetypes-test.adoc[]
```

Comment:
can you rename the common snippets to something like `common-junit-params.adoc` so that it is easy to spot and reuse in other guides. ?

Reply:
Just did, please review again, thanks